src/vaadin-radio-group.html

```diff
@@ -276,6 +277,7 @@
               e.preventDefault();
               this._selectIncButton(!horizontalRTL, checkedRadioButton);
             }
+            this._checkKeyDown = true;
```

Comment:
This flag doesn't seem to be used anywhere, please remove.

Reply:
removed